macOS/DuckDuckGo/DefaultBrowserAndAddToDockPrompts/BannerView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  BannerView.swift
```

Comment:
I didn’t add any prefix related to SAD and ATT for this particular view, given that it is pretty generic and can be reused for other banners we might implement. 

I have an idea of implementing a banner coordinator that will serve as a proxy for other banners we would like to show.